src/Tribe/Assets.php

```diff
@@ -504,12 +491,47 @@ public function register( $origin, $slug, $file, $deps = array(), $action = null
 		$this->assets[ $slug ] = $asset;
 
 		// Sorts by priority
-		uasort( $this->assets, array( $this, 'order_by_priority' ) );
+		uasort( $this->assets, [ $this, 'order_by_priority' ] );
 
 		// Return the Slug because it might be modified
 		return $asset;
 	}
 
+	/**
+	 * Parse the localize argument for a given asset object.
+	 *
+	 * @since TBD
+	 *
+	 * @param  stdClass $asset Argument that set that asset
+	 *
+	 * @return stdClass
+	 */
+	public function parse_argument_localize( stdClass $asset ) {
+		if ( empty( $asset->localize ) ) {
+			return $asset;
+		}
+
+		if ( ! is_array( $asset->localize ) && ! is_object( $asset->localize ) ) {
```

Comment:
This could lower the cognitive load by using an OR negation: `if ( ! ( is_array( $asset->localize ) || is_object( $asset->localize ) ) )